### PR TITLE
Create dashboard page

### PR DESCRIPTION
**Pull request by Agata-Duda**
Feature: Create dashboard page

- Added a new background vector for the dashboard.
- Created a new fragment for dashboard.
- Created redirect function for dashboard.
- Redirect will be completed within the redirect task.
- Added Dashboard to navigation.
- Added string value for Board title.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -12,4 +12,7 @@
     <string name="error">Error!</string>
     <string name="register">Register</string>
     <string name="fullName">Full Name</string>
+    <string name="boardText">Board</string>
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```
Comment (psavarmattas) [inline app/src/main/res/values/strings.xml:17]: These are not needed and are redundant code.

File: app/src/main/java/com/taskmuse/app/ui/activity/MainActivity/MainActivity.java [hunk 2]
```diff
@@ -57,4 +57,11 @@ public void alreadyUser(){
                 .addToBackStack(null) // Name can be null
                 .commit();
     }
+    public void redirectToDashboard(){
+        fragmentManager.beginTransaction()
+                .replace(R.id.fragmentContainerView, Login_Fragment.class, null)
```
Comment (psavarmattas) [inline app/src/main/java/com/taskmuse/app/ui/activity/MainActivity/MainActivity.java:62]: Here `Login_Fragment.class` should not be there I suppose as you are using this function to go to Dashboard. Correct? Or is it temporary?

File: .idea/assetWizardSettings.xml
Comment (psavarmattas) [inline .idea/assetWizardSettings.xml:1]: What is this file used for?

**Review by psavarmattas — CHANGES_REQUESTED** [review]

File: .idea/assetWizardSettings.xml
Comment (Agata-Duda) [reply .idea/assetWizardSettings.xml:1]: Generated automatically after vector creation

File: app/src/main/java/com/taskmuse/app/ui/activity/MainActivity/MainActivity.java (hunk 2 above)
Comment (Agata-Duda) [reply app/src/main/java/com/taskmuse/app/ui/activity/MainActivity/MainActivity.java]: fixed - bad copy paste

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (Agata-Duda) [reply app/src/main/res/values/strings.xml]: Done

**Comment by psavarmattas** [comment]
LGTM🫡

**Review by psavarmattas — APPROVED** [review]

**Review by kajal1106 — APPROVED** [review]
Looks good to me!

**Pull request merged**